tests/event-bus/e2e-tester/Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY e2e-tester.go .
 RUN ls ./
 RUN CGO_ENABLED=0 GOOS=linux go build -v -a -installsuffix cgo -o e2e-tester .
 
-FROM scratch
+FROM curlimages/curl:7.66.0
```

Comment:
This image contains `curl` and shell, which fits our requirement.

Reply:
Was there any reason not to depend on `alpine` directly and adding a simple `apk add --no-cache curl`?

Reply:
@antoineco This is an official curl image and it is based on alpine anyway. Using this image has the advantage of pinning to exact curl version. In alpine, you get the version that is supported by maintainers.